demo/js/demo.js

```diff
@@ -97,6 +97,10 @@ export class App extends Component {
             page >= 1 && page <= pageCount ? `/page/${page}` : '#'
           }
           hrefAllControls
+          showFirstLabel={true}
```

Comment:
Actually just `showFirstLabel` and `showLastLabel` is enough (default to boolean `true`, no need for `={true}`.

The rest is perfect. I'll certainly split the Props doc for the v9 (by categories: general, theming/styling, last/first button), because our table starts to be hard to read.

Reply:
Yeah that's true, removed